.browserslistrc

```diff
@@ -1,7 +1,11 @@
-[production staging]
-> 1%
-ie 10
+# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+#
+# For IE 10-11 support, please remove 'not' from the last line of the file and adjust as needed
 
-[development]
-last 1 chrome version
-last 1 firefox version
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+IE 10-11
```

Comment:
please keep the distinction between the envs and leave a lightweight configuration for the devs. also, i would suggest to drop IE 10 from the list.

Reply:
> i would suggest to drop IE 10 from the list.

That part is done via updated